src/System.DirectoryServices/tests/System/DirectoryServices/ActiveDirectory/DomainControllerTests.cs

```diff
@@ -42,7 +42,11 @@ public void GetDomainController_InvalidName_ThrowsActiveDirectoryObjectNotFoundE
         public void GetDomainController_InvalidIPV6_ThrowsActiveDirectoryObjectNotFoundException()
         {
             var context = new DirectoryContext(DirectoryContextType.DirectoryServer, "[::1]:port");
-            Assert.Throws<ActiveDirectoryObjectNotFoundException>(() => DomainController.GetDomainController(context));
+            Exception exception = Record.Exception(() => DomainController.GetDomainController(context));
+            Assert.NotNull(exception);
+            Assert.True(exception is ActiveDirectoryObjectNotFoundException ||
+                        exception is ActiveDirectoryOperationException,
+                        $"We got unrecognized exception {exception}");
```

Comment:
There are 3 additional tests failing in nightly runs -- see [20180123.01](https://mc.dot.net/#/product/netcore/master/source/official~2Fcorefx~2Fmaster~2F/type/test~2Ffunctional~2Fcli~2F/build/20180123.01/workItem/System.DirectoryServices.Tests) and [20180123.04](https://mc.dot.net/#/product/netcore/master/source/official~2Fcorefx~2Fmaster~2F/type/test~2Ffunctional~2Fcli~2F/build/20180123.04/workItem/System.DirectoryServices.Tests)

Reply:
will take a look and update the PR. Thanks @karelz